src/client/Client.h

```diff
@@ -1964,6 +1968,7 @@ class Client : public Dispatcher, public md_config_obs_t {
 
   ceph::coarse_mono_time last_auto_reconnect;
   std::chrono::seconds caps_release_delay, mount_timeout;
+  int injected_write_delay_secs;
```

Comment:
I'm in favour of a fixed-width unsigned int over here maybe uint32_t if not unint64_t

Reply:
Why? Mind explaining a bit? The min value from the config is 0.

Reply:
its more precise, its not platform dependent and this is tracking seconds i.e. time so doesn't make sense to be signed int.

Reply:
Theoretical your are correct. But, I don't see much point in moving this to an unsigned int, especially since its test a test case only config to be executed in a controlled environment.

Reply:
sure, it was anyways a nitpicking so no worries